.env

```diff
@@ -1 +1 @@
-REACT_APP_ENDPOINT_URL='https://crudcrud.com/api/59034e8c8fad4bcaaf8c81a168e22d03/tasks'
\ No newline at end of file
+REACT_APP_CRUD_ENDPOINT_URL='https://crudcrud.com/api/59034e8c8fad4bcaaf8c81a168e22d03/tasks'
```

Comment:
you should remove REACT_APP_

Reply:
can't remove this as it is necessary for the identification of an environment var